nikola/data/themes/base/templates/list.tmpl

```diff
@@ -8,8 +8,11 @@
     </header>
     %if items:
     <ul class="postlist">
-    % for text, link in items:
+    % for text, link, count in items:
```

Comment:
other lists might not contain the count thing, you should look into that (in `self.site.generic_post_list_renderer`?)